src/openrct2/world/Footpath.cpp

```diff
@@ -1940,9 +1942,10 @@ void footpath_update_path_wide_flags(const CoordsXY& footpathPos)
             && !std::get<3>(pathList)->AsPath()->IsWide())
         {
             constexpr uint8_t edgeMask1 = EDGE_SE | EDGE_SW;
-            if ((pathConnections & FOOTPATH_CONNECTION_NE) && std::get<7>(pathList) != nullptr
-                && !std::get<7>(pathList)->AsPath()->IsWide()
-                && (std::get<7>(pathList)->AsPath()->GetEdges() & edgeMask1) == edgeMask1 && std::get<0>(pathList) != nullptr
+            const auto& path_element1 = std::get<7>(pathList);
```

Comment:
```suggestion
            const auto* pathElement7 = std::get<7>(pathList);
```
Makes it more clear that its the 7th element in the list.